starters/next-chakra-ui/src/pages/fetch-example.tsx

```diff
@@ -36,7 +36,7 @@ const FetchExample: NextPage = () => {
           borderBottom="4px solid #2464ec"
           mt={8}
         >
-          NextJS 12 fetching data example from a REST API
+          NextJS fetching data example from a REST API
```

Comment:
The text for `Greeting.test.tsx` needs to be updated too because the test is currently failing.